package.json

```diff
@@ -24,6 +24,8 @@
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
     "@faker-js/faker": "^7.6.0",
+    "@material-ui/core": "^4.12.4",
```

Comment:
are you sure these are the same versions as we're using in the backyard and website?

Reply:
Yes Checked